core/markup/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
-  "compilerOptions": {
-    "outDir": "./lib"
-  },
   "include": [
     "./src/**/*.ts"
   ],
+  "exclude": [
```

Comment:
this one as well. Don't think it's needed.

Reply:
should i just rmeove this from all tsconfigs then? or just add this to the the base?

Reply:
Is it needed? Since we're specifying `./src/` explicitly I didn't think this would be required in any of our configs.

Reply:
Not needed, https://www.typescriptlang.org/tsconfig#exclude, will remove across entire repo and move the include into the base and people can still override that if theyd like

Reply:
nvm will not be moving include into the base, theres some build errors with that